### PR TITLE
refactor: add // or /*...*/ to make example be highlighted correctly

### PR DESCRIPTION
**Pull request by danielpeintner**
Makes the examples and code snippets highlight correctly..

There are examples (see [Example 35](https://w3c.github.io/wot-thing-description/#example-35) or [Example 41](https://w3c.github.io/wot-thing-description/#dataschema-serialization-sample)) were I had to add an outer `{ }` to make it a valid JSON. This is _not_ completely correct but I think it is still fine.

fixes https://github.com/w3c/wot-thing-description/issues/1701

***
<a href="https://pr-preview.s3.amazonaws.com/danielpeintner/wot-thing-description/pull/1711.html" title="Last updated on Oct 19, 2022, 12:06 PM UTC (8c7d8a9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1711/57bf285...danielpeintner:8c7d8a9.html" title="Last updated on Oct 19, 2022, 12:06 PM UTC (8c7d8a9)">Diff</a>

### CODE REVIEW

File: index.template.html [hunk 1]
```diff
@@ -5669,8 +5696,8 @@ <h2>Examples</h2>
              }
            ],
            "properties" : {
-                "onOff : {
-                    "type: "boolean",
+                "onOff": {
+                    "type": "boolean",
```
Comment (danielpeintner) [inline index.template.html:5700]: Note: It fixes also some snippets that were not correct before...

**Review by relu91 — APPROVED** [review]
LGTM, however the "with defaults" snippets are still broken, right?

**Comment by danielpeintner** [comment]
> LGTM, however the "with defaults" snippets are still broken, right?

This is correct and is fixed by https://github.com/w3c/wot-thing-description/pull/1700

**Review by sebastiankb — APPROVED** [review]

**Comment by sebastiankb** [comment]
from today's TD call decided to merge

**Pull request merged**